src/extractors/custom/www.ftchinese.com/index.test.js

```diff
@@ -0,0 +1,107 @@
+import assert from 'assert';
+import fs from 'fs';
```

Comment:
The web test is failing b/c of this line. If you can move the `fs` import to a require (`const fs = require('fs');`) that should take care of that. It looks like I had forgotten to update the parser generator. Fixing that in another PR.